plugin/web/extensions/sidebar/core/components/hooks.ts

```diff
@@ -136,7 +136,7 @@ export default () => {
       });
 
       // const options = data?.find(d => d.id === dataset.id)?.components;
-      postMsg({ action: "addDatasetToScene", payload: { dataset } });
+      postMsg({ action: "addDatasetToScene", payload: { dataset: { ...dataset } } });
```

Comment:
Can you explain why you needed this changed to spreading the dataset?

Reply:
These three comments are a merge mistake.:pray:
I fixed in this commit.
[fix: a merge mistake](https://github.com/eukarya-inc/reearth-plateauview/pull/119/commits/214a9c85fe4f8cdd880ad5e631008385b7c30a6f)